README.md

```diff
@@ -0,0 +1,36 @@
+# Design first page  portfolio app based in a figma template 
+
+> Skeleton design of the first page of a mobile app in html and css based on a figma template
+
+> Properties and values of many of the properties in the css file has been extracted from the figma template.
+
+> Image has been exported from the figma template. When a group of images in the figma template forms a group this is exported as a svg image.
```

Comment:
Kindly keep the description to one or two lines that describe your project briefly as described in the [Readme Template](https://github.com/microverseinc/readme-template) 😃

Reply:
- [ ] [OPTIONAL] kindly make the description brief as requiredd by the first reviewer but it should be optional
 

> Kindly keep the description to one or two lines that describe your project briefly as described in the [Readme Template](https://github.com/microverseinc/readme-template) smiley